content/post/36-a-look-at-windows.md

```diff
@@ -0,0 +1,52 @@
+
+Now, the building experience should be consistent with other platforms, in addition the [documentation has been rewritten](https://github.com/ipfs/go-ipfs/blob/master/docs/windows.md), clarifying the process and adding a section that covers Windows specific concerns and how to deal with them.
+
+### Access errors with temporary files
+![garbage](img/garbage.png)  
+Previously, when trying to move data blocks from a temporary location, the destination path was often mangled. This led to "Access Denied" errors and the creation of garbage files. For deeper details, see the [Windows initiative issue](https://github.com/ipfs/go-ipfs/issues/4808#issuecomment-375796905).
+
+Now, operations should succeed as expected.
+
+### File output names
+{TODO:[MD] numbered list + headers don't render correctly on ipfs.io}  
+Previously, there was no filtering applied when extracting files out of IPFS. This caused multiple issues.  
+1. #### Extracting content from IPFS could fail due to native file system restrictions
+![filenames](img/filenames-crop.png)  
+FAT32, NTFS, ReFS, and Windows itself, all impose their own path limitations. If a hash contained a non-legal path, operations like `ipfs get` would fail. In the image above, the [XKCD archive ](https://github.com/ipfs/archives/issues/21) could not be downloaded on Windows because the directory "1031 - s keyboard leopard " ends in a space.  
+
+2. #### Hashes that contained platform-specific malicious files and symlinks could be extracted
+![overwrite 1](img/overwrite%201.png)  
+![overwrite 2](img/overwrite%202.png)  
+{TODO: This is a minor(currently impractical and easily detected) security concern but a security concern nonetheless, consider what tone to use here}
+
+Previously, it was possible for users to craft specific hashes that could escape the extraction root and overwrite files, if the target file's location was known in advanced. In addition, symlinks whose target escaped the hash-root (either via relative or absolute means) were allowed to be created.
+
+Now, we account for these foreign limitations and translate pathnames into platform legal paths that are restricted to their root.
+
+### `stdio`
+![pipe](img/pipe.png)  
+Previously, stdin support was explicitly disabled in `go-ipfs`, preventing basic [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) with other applications.
+Now, stdin support has been added to the Windows version of go-ipfs which allows you to place ipfs anywhere in a [pipeline](https://en.wikipedia.org/wiki/Pipeline_(Unix)).  
+
+***
+
+There’s still a lot more to improve for IPFS on Windows. If you’re interested in contributing (or just keeping an eye on progress), check the [Windows initiative issue](https://github.com/ipfs/go-ipfs/issues/4808) on `go-ipfs`.
```

Comment:
Expand this section a LOT. For reference, the bottom of this blog post from last week is a good example ("Want to contribute?" / "Do you have questions?" sections): https://ipfs.io/blog/35-ipfs-companion-2-2-0/

Reply:
Note that last week's contribute section is bit browser-centric. There are good "generic" points in earlier posts, eg. https://ipfs.io/blog/33-js-ipfs-0-28/ has:

> - Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/pm/#all-hands-call
> - Hack with IPFS and show us what you made! The All Hands call is also the perfect venue for demos, join in and show us what you built
> - Join the discussion at http://discuss.ipfs.io/ and help users finding their answers.